tests/test-data/test_configs/default/blocklist.txt

```diff
@@ -0,0 +1,3 @@
+baddomain.com.
+foo.com
+*.foo.com.
```

Comment:
when testing this, it would be good to see if `bar.baz.foo.com` also matches the wildcard.

Reply:
The tests cover this (specifically the www.com.foo.com lookup.)